operations/model.go

```diff
@@ -20,11 +20,12 @@ import (
 const localConfigPath = ".evergreen.local.yml"
 
 type ClientProjectConf struct {
-	Name     string   `json:"name" yaml:"name,omitempty"`
-	Default  bool     `json:"default" yaml:"default,omitempty"`
-	Alias    string   `json:"alias" yaml:"alias,omitempty"`
-	Variants []string `json:"variants" yaml:"variants,omitempty"`
-	Tasks    []string `json:"tasks" yaml:"tasks,omitempty"`
+	Name           string   `json:"name" yaml:"name,omitempty"`
+	Default        bool     `json:"default" yaml:"default,omitempty"`
+	Alias          string   `json:"alias" yaml:"alias,omitempty"`
+	Variants       []string `json:"variants" yaml:"variants,omitempty"`
+	Tasks          []string `json:"tasks" yaml:"tasks,omitempty"`
+	IncludeWorking bool     `json:"include_working" yaml:"include_working,omitempty"`
```

Comment:
yaml tag,  I think, and maybe called "patch_uncommitted_changes"?

Reply:
renamed the field.
I thought there was a yaml tag already (?)